app/components/resource/ResourcesTable.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component, PropTypes } from 'react';
+import Table from 'react-bootstrap/lib/Table';
+import { connect } from 'react-redux';
+
+import ResourcesTableItem from 'components/resource/ResourcesTableItem';
+import { getCurrentReservation, getNextReservation } from 'utils/DataUtils';
+
+export class UnconnectedResourcesTable extends Component {
+  constructor(props) {
+    super(props);
+    this.renderResourcesTableItem = this.renderResourcesTableItem.bind(this);
```

Comment:
- [ ] This shouldn't be necessary with the `renderResourcesTableItem`. It's not getting anything from `this`.